channeldb/invoices.go

```diff
@@ -433,6 +433,65 @@ func fetchInvoiceNumByRef(invoiceIndex, payAddrIndex, setIDIndex kvdb.RBucket,
 	}
 }
 
+// FetchPendingInvoices returns all invoices that have not yet been settled or
+// canceled. The returned map is keyed by the payment hash of each respective
+// invoice.
+func (d *DB) FetchPendingInvoices(_ context.Context) (
+	map[lntypes.Hash]invpkg.Invoice, error) {
+
+	result := make(map[lntypes.Hash]invpkg.Invoice)
+
+	err := kvdb.View(d, func(tx kvdb.RTx) error {
+		invoices := tx.ReadBucket(invoiceBucket)
+		if invoices == nil {
+			return invpkg.ErrNoInvoicesCreated
+		}
```

Comment:
should we return nil here for the same reason as below perhaps? for the case where there just is no invoice yet? I see the same is done in other methods like `InvoicesSettledSince` where it is not an error if there are no invoices yet